install/kubernetes/Makefile.digests

```diff
@@ -2,12 +2,12 @@
 # Copyright 2022 Authors of Cilium
 # SPDX-License-Identifier: Apache-2.0
 
-CILIUM_DIGEST := "sha256:adbb60f9d19d641a5496fcfc4769ebb55c1d4cd7699fef752f7c94754194cf83"
```

Comment:
same comment as the one for 1.11 release. Is this intended?

Reply:
Yes it's intentional, see https://github.com/cilium/cilium/pull/21665#discussion_r992757685